diesel/src/sqlite/connection/mod.rs

```diff
@@ -173,6 +173,16 @@ impl SqliteConnection {
         self.transaction_sql(f, "BEGIN EXCLUSIVE")
     }
 
+    /// Enables or disables the `load_extension` SQL function.
+    ///
+    /// See the documentation for the [`enable_load_extension` C function] for
+    /// details.
+    ///
+    /// [`enable_load_extension` C function]: https://www.sqlite.org/c3ref/enable_load_extension.html
+    pub fn enable_load_extension(&self, enabled: bool) -> QueryResult<()> {
```

Comment:
I think it would result in a clearer API if we have two functions here, instead of one with a boolean flag.

Reply:
Added a separate method `disable_load_extension` for sqlite connection api